apps/demo/lib/config.ts

```diff
@@ -40,24 +33,12 @@ if (!projectId) {
 // Networks
 type AppKitNetworksType = [AppKitNetwork, ...AppKitNetwork[]]
 
-export const evmNetworks = [
-  mainnet,
-  optimism,
-  polygon,
-  zksync,
-  arbitrum,
-  base,
-  baseSepolia,
-  unichainSepolia,
-  berachain,
-  sepolia,
-  gnosis,
-  hedera,
-  aurora,
-  mantle
-] as [AppKitNetwork, ...AppKitNetwork[]]
+export const evmNetworks = [mainnet, optimism, bsc, polygon, avalanche, arbitrum, zksync, base] as [
+  AppKitNetwork,
+  ...AppKitNetwork[]
+]
 
-export const solanaNetworks = [solana, solanaTestnet, solanaDevnet] as AppKitNetworksType
+export const solanaNetworks = [solana, solanaDevnet] as AppKitNetworksType
```

Comment:
why?

Reply:
It was previously like that, since we are showing limited networks to the users, it's better to keep the same array on background. Just reverting